concepts/structs/about.md

```diff
@@ -39,6 +41,37 @@ fmt.Printf("name: %s age: %d\n", person.name, person.age)
 // Output: name: age: 0
 ```
 
+Another way of creating a new instance of a struct is by using the `new` built-in: 
+```go
+s2 := new(Shape)
+fmt.Printf("name: %s size: %d\n", s2.name, s2.size)
+// Output: name:  size: 0
+```
+
+Sometimes it can be nice to have a _constructor_, which is a function to create an instance of a struct.
+The following constructor can be used to create a new instance of `Shape` and automatically set a default-value for the size of the shape:
```

Comment:
```suggestion
Sometimes it can be nice to have functions that help us create struct instances. By convention, these functions usually are called `New` or have their names starting with `New`, but since they are just regular functions, you can give them any name you want.  They might remind you of constructors in other languages, but in Go they are just regular functions.

In the following example, one of these `New` functions is used to create a new instance of `Shape` and automatically set a default value for the `size` of the shape:
```

Avoided the word "constructor" here because they are not really constructors in the sense they are "attached" to a particular type. I think it's important to empathize that these are just regular functions.